Introduction.playground/Pages/6. Memory Management - Part Two.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,53 @@
+//: [Previous](@previous)
+
+import Foundation
+import RxSwift
+
+//: # Memory Management - Part Two
+//:
+//: In previous example we learned that certain sequence observers are automatically disposed when a sequence completes.
+//:
+//: You probably wonder - what if I have a sequence that never completes? Or a sequence that won't complete before my object should be deallocated?
+//: In this case we need to manually dispose of the sequence observer. There are two ways of doing this.
```

Comment:
*manually* suggests that the `dispose` is called by ourselves. Which is true, but only in the first case. Maybe `In this case the subscription disposal must be taken care of by ourselves`?